commit 05e69f495f2a23b294601720962775c0cf617e76 (refs/changes/02/3302/2)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2018-02-06T17:33:57+01:00 (1 year, 8 months ago)
    
    OS-6498 bnxe does not attach on debug kernels
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/usr/src/uts/common/io/bnxe/bnxe_debug.c b/usr/src/uts/common/io/bnxe/bnxe_debug.c
index 1411045f2c..e8587a17b3 100644
--- a/usr/src/uts/common/io/bnxe/bnxe_debug.c
+++ b/usr/src/uts/common/io/bnxe/bnxe_debug.c
@@ -186,7 +186,7 @@ void BnxeLogWarn(void * pDev,
 }
 
 
-#ifdef DBG
+#ifdef DEBUG
 
 void BnxeLogDbg(void * pDev,
                 char * pFmt,
@@ -209,7 +209,7 @@ void BnxeLogDbg(void * pDev,
     cmn_err(CE_WARN, "!%s: %s", BnxeDevName(pUM), buf);
 }
 
-#endif /* DBG */
+#endif /* DEBUG */
 
 
 void BnxeDumpMem(um_device_t * pUM,
